index.js

```diff
@@ -0,0 +1,35 @@
+import Book from './modules/classes.js';
+
+const bookTitle = document.querySelector('#title');
+const theForm = document.querySelector('form');
+const bookAuthor = document.querySelector('#author');
+const listBtn = document.querySelectorAll('.listBtn');
+const addActive = document.querySelectorAll('.section');
+
+theForm.addEventListener('submit', () => {
+  const newBook = new Book(bookTitle.value, bookAuthor.value);
+  newBook.addBook();
+});
+
+Book.displayUI();
+const removeButton = document.querySelectorAll('.remove-btn');
+removeButton.forEach((item) => item.addEventListener('click', function () {
+  const deleteBook = new Book(bookTitle.value, bookAuthor.value, this.id);
+  deleteBook.removeBook();
+}));
```

Comment:
- [ ] I recommend using arrow function as the callback for an event listener: 
```
removeButton.forEach((item) => item.addEventListener('click',  () =>  {
  const deleteBook = new Book(bookTitle.value, bookAuthor.value, this.id);
  deleteBook.removeBook();
}))
```

Reply:
Hello, has this suggestion worked for you !!!

Reply:
Yes it did, do you have any issues with it?